dashboard/src/components/ui/table.tsx

```diff
@@ -66,6 +67,34 @@ const TableRow = React.forwardRef<
 ));
 TableRow.displayName = 'TableRow';
 
+type LinkProps = RouterLinkProps & { className?: string }
+
+type TableRowWithLinkAttributes = React.TdHTMLAttributes<HTMLTableRowElement> & {
+  linkProps: LinkProps
+}
+
+const TableRowWithLink = React.forwardRef<
+  HTMLTableRowElement,
+  TableRowWithLinkAttributes
+>(({ className, children, linkProps, ...props }, ref) => (
+  <tr
+    ref={ref}
+    className={cn(
+      'border-b transition-colors hover:bg-slate-100/50 data-[state=selected]:bg-slate-100 dark:hover:bg-slate-800/50 dark:data-[state=selected]:bg-slate-800',
+      className,
+    )}
+    {...props}
+  >
+    <Link
+      className={cn('flex flex-1')}
+      {...linkProps}
+    >
+        {children}
+      </Link>
+  </tr>
```

Comment:
The child of a `tr` should be a `td`, <a> is not valid html

Reply:
![image](https://github.com/user-attachments/assets/439068ce-54e3-46f2-8a36-70acd021ad66)

its creating this console warning (yes we have some other warnings for invalid html that you dind't create, we will need to fix those another day also)